### PR TITLE
Feature/issue92

### PR DESCRIPTION
**Pull request by Hanyang0901**
Fix validators for real data of the schema.

### CODE REVIEW

File: MultCo-ISD-API/MultCo-ISD-API/V1/Validators/ServiceV1DTOValidator.cs [hunk 1]
```diff
@@ -17,38 +17,31 @@ public ServiceV1DTOValidator()
...
 
             RuleFor(x => x.CustomerConnectMethod)
-                .NotNull().WithMessage("Customer Connect Method cannot be null")
-                .MaximumLength(50).WithMessage("Customer Connect Method cannot exceed 50 chars");
+                .MaximumLength(255).WithMessage("Customer Connect Method cannot exceed 255");
 
-            RuleFor(x => x.ExpirationDate)
-                .NotNull().WithMessage("Expiration Date cannot be null");
+            //RuleFor(x => x.ExpirationDate); 
+                //.NotNull().WithMessage("Expiration Date cannot be null");
```
Comment (Gubriels) [inline MultCo-ISD-API/MultCo-ISD-API/V1/Validators/ServiceV1DTOValidator.cs:44]: There a reason this is commented out instead of removed?

**Comment by maxjaspersmiley** [comment]
I've built this and can't get PUT to work. Maybe we can chat on Slack sometime soon. Sorry it took me so long to get to!

**Comment by Gubriels** [comment]
Can we get this merged tonight? @mohamm-alsaid @Hanyang0901 @maxjaspersmiley

**Comment by Gubriels** [comment]
@maxjaspersmiley We got put to work. There was some conflicting code. @mohamm-alsaid Can you tag max in a comment once you commit it?

**Pull request merged**